app/viewmodels/ImportVatViewModel.scala

```diff
@@ -0,0 +1,266 @@
+  )(implicit messages: Messages, config: AppConfig): ImportVatViewModel = {
+
+    val hasRequestedCertificates: Boolean = certificatesForAllEoris.exists(_.requestedCertificates.nonEmpty)
+    val hasCurrentCertificates: Boolean   = certificatesForAllEoris.exists(_.currentCertificates.nonEmpty)
+
+    ImportVatViewModel(
+      title = Some(messages("cf.account.vat.title")),
+      backLink = Some(config.customsFinancialsFrontendHomepage),
+      heading = populateHeading(),
+      certificateAvailableGuidance = populateCertsAvailableGuidance(),
+      last6MonthsH2Heading = populateLast6MonthsH2Heading(),
+      notificationPanel = if (hasRequestedCertificates) Some(populateNotificationPanel()) else None,
+      currentStatements = if (hasCurrentCertificates) populateCurrentStatements(certificatesForAllEoris) else Seq.empty,
+      currentStatementsNotAvailableGuidance =
+        if (hasCurrentCertificates) None else Some(populateCurrentStatNotAvailableGuidance),
+      certsOlderThan6MonthsGuidance = populateCertsOlderThan6MonthsGuidance(serviceUnavailableUrl),
+      chiefDeclarationGuidance = populateChiefDeclarationGuidance,
+      helpAndSupportGuidance = populateHelpAndSupportGuidance
+    )
+  }
+
+  private def populateHeading(implicit messages: Messages): HtmlFormat.Appendable =
+    h1Component("cf.account.vat.title", id = Some("import-vat-certificates-heading"), classes = "govuk-heading-xl")
+
+  private def populateCertsAvailableGuidance(implicit messages: Messages): HtmlFormat.Appendable = pComponent(
+    "cf.account.vat.available-text"
+  )
+
+  private def populateLast6MonthsH2Heading(implicit messages: Messages): HtmlFormat.Appendable = h2Component(
+    "cf.account.vat.your-certificates.heading"
+  )
+
+  private def populateNotificationPanel(implicit messages: Messages, appConfig: AppConfig): HtmlFormat.Appendable = {
+    val htmlContent: Html = HtmlFormat.fill(
+      Seq(
+        linkComponent(
+          linkMessage = messages("cf.import-vat.requested-certificates-available-link-text"),
+          location = appConfig.requestedStatements(C79Certificate),
+          preLinkMessage = Some(messages("cf.account.detail.requested-certificates-available-text.pre")),
+          postLinkMessage = Some(messages("cf.account.detail.requested-certificates-available-text.post")),
+          pClass = "govuk-body govuk-!-margin-bottom-1"
+        )
+      )
+    )
+
+    divComponent(
+      content = htmlContent,
+      classes = Some("notifications-panel"),
+      id = Some("notification-panel")
+    )
+  }
+
+  private def populateCurrentStatements(
+    certsForAllEoris: Seq[VatCertificatesForEori]
+  )(implicit msgs: Messages): Seq[ImportVatCurrentStatementRow] =
+    certsForAllEoris.indices
+      .map { historyIndex =>
+        if (certsForAllEoris(historyIndex).currentCertificates.nonEmpty) {
+
+          val eoriHeading: Option[HtmlFormat.Appendable] = populateEoriHeading(certsForAllEoris, historyIndex)
+
```

Comment:
this empty line seems unnecessary to me?

Reply:
Done